requirements.txt

```diff
@@ -2,7 +2,7 @@ black==22.10.0
 build==0.9.0
 flake8==6.0.0
 python-iso639==2022.11.27
-mypy==0.991
+mypy==1.1.1
```

Comment:
Had to upgrade the mypy version, since the mypy run at the CircleCI build failed at the previous commit (e33f8c6605c9a89480f4e9a2dd4090534a13c164).